help/saml-authentication.md

```diff
@@ -177,6 +178,33 @@ providers.
 
     Your Keycloak realm URL will look something like this: `https://keycloak.example.com/auth/realms/yourrealm`.
 
+{tab|auth0}
+
+{!upgrade-to-plus-if-needed.md!}
+
+1. Set up SAML authentication by following [Auth0's documentation](https://auth0.com/docs/authenticate/protocols/saml/saml-sso-integrations/configure-auth0-saml-identity-provider#configure-saml-sso-in-auth0)
+   to create a new application. You don't need to save the certificates or other information detailed.
+   All we will need is the **SAML Metadata URL**.
```

Comment:
Let's change to:

> The only information you will need about this application is the **SAML Metadata URL**. You don't need to save the certificates or other details.

(note: We avoid using the first person ("we") in the help center docs.)